pkgs/applications/misc/audio/sox/default.nix

```
@@ -39,8 +39,9 @@ stdenv.mkDerivation rec {
   pname = "sox";
   version = "unstable-2021-05-09";
 
-  src = fetchzip {
-    url = "https://sourceforge.net/code-snapshots/git/s/so/sox/code.git/sox-code-42b3557e13e0fe01a83465b672d89faddbe65f49.zip";
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/sox/code";
```

Would make this change a no-op and thus saving us the mass-rebuild :)

```suggestion
  src = fetchgit {
    name = "source";
    url = "https://git.code.sf.net/p/sox/code";
```